packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -29,9 +28,9 @@ export interface IJSONFormatProps extends ITruncatedFormatProps {
     stringify?: (obj: any) => string;
 }
 
-@PureRender
-export class JSONFormat extends React.Component<IJSONFormatProps, {}> {
+export class JSONFormat extends React.PureComponent<IJSONFormatProps, {}> {
```

Comment:
@michael-yx-wu are you gonna make a PR for those `PureRender`s? If not, mind filing an issue with the details? 👌